Common/API/LiveAlgorithmResults.cs

```diff
@@ -0,0 +1,208 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+using QuantConnect.Api;
+using QuantConnect.Orders;
+
+namespace QuantConnect.API
+{
+    /// <summary>
+    /// Details a live algorithm from the "live/read" Api endpoint
+    /// </summary>
+    public class LiveAlgorithmResults : RestResponse
+    {
+        /// <summary>
+        /// Represents data about the live running algorithm returned from the server
+        /// </summary>
+        public LiveResults LiveResults { get; set; }
+    }
+
+    /// <summary>
+    /// Holds information about the state and operation of the live running algorithm
+    /// </summary>
+    public class LiveResults
```

Comment:
Can we potentially reuse the "Common\Packets\LiveResult" class here? BacktestResult and LiveResult are almost identical in theory on the view side. Reusing the same classes would help future view implementations.

Potentially we could even do a slightly reshuffle to make the classes extended from a "AlgorithmResult" base common implementation.